mycore-base/src/main/java/org/mycore/tools/MCRLanguageOrientationHelper.java

```diff
@@ -0,0 +1,19 @@
+package org.mycore.tools;
+
+
+import java.util.Set;
+
+public class MCRLanguageOrientationHelper {
+
+    static Set<String> RTL_LANGUAGES = getRTLLanguages();
+
+    private static Set<String> getRTLLanguages() {
+        return Set.of("ar", "dv", "fa", "ha", "he", "iw", "ji", "ps", "sd", "ug", "ur", "yi");
```

Comment:
is this list complete? - use property for easier configuration / extendability

Reply:
i extracted them to properties